godot-core/src/obj/script.rs

```diff
@@ -153,17 +154,28 @@ pub trait ScriptInstance: Sized {
     /// The engine may call this function if
     /// [`IScriptExtension::is_placeholder_fallback_enabled`](crate::classes::IScriptExtension::is_placeholder_fallback_enabled) is enabled.
     fn property_set_fallback(this: SiMut<Self>, name: StringName, value: &Variant) -> bool;
+
+    /// This function will be called to handle calls to [`Òbject::get_method_argument_count`](crate::classes::Object::get_method_argument_count)
+    /// and `Callable::get_argument_count`.
+    ///
+    /// If [`None`] is returned the public methods will return `0`.
```

Comment:
```suggestion
    /// If `None` is returned, the public methods will return `0`.
```

`None`  is so ubiquitious, there's no point in linking to its docs 🙂